### DISCUSSION TITLE
Blog article and sample app 'How to write an application that integrates Kratos in Go'

### DISCUSSION THREAD

**Discussion by davidoram** [ory/kratos#1224] [Show and tell]
I've written a [blog post](https://stories.abletech.nz/integrating-third-party-provider-kratos-f5514b53af66) with accompanying [source](https://github.com/davidoram/kratos-selfservice-ui-go) that describes a sample go app that integrates with [Kratos](https://www.ory.sh/kratos/)

**Comment by vinckr** [comment]
I love it 😍 . 
Thanks a ton for the detailed article!
Will test it out in the coming days and send it to some people, who have been looking for something like this.

**Reply by davidoram** [reply]
Thanks @vinckr appreciate the kind words.  If you have any feedback/errors/improvements let me know and I will do my best to update. 

Cheers
Dave
